tools/travis/before_install.sh

```diff
@@ -29,8 +29,19 @@ export WHEELHOUSE
 export DISPLAY=:99.0
 # This causes way too many internal warnings within python.
 # export PYTHONWARNINGS="d,all:::skimage"
-# codecov will combine different paths
-export TEST_ARGS="--doctest-modules --cov=skimage"
+
+# Doctest's don't run well on other's systems because of the
```

Comment:
What does this mean?

Reply:
I think I was able to fix this by moving the conftest file in the `skimage` directory.